source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.csse3200.game.ui.DialogueBox;
```

Comment:
Is this class meant to be included in this pull request? It doesn't seem to actually be used in the feature or be a part of the description of this PR.